samples/api-doc-search/azure-function-nodejs/src/functions/BlobTriggerEventGrid.js

```diff
@@ -0,0 +1,61 @@
+// Importing necessary modules
+// 'app' is imported from '@azure/functions' to create and manage Azure functions
+const { app } = require('@azure/functions');
+
+// 'InitiateEmbeddings' is  imported from 'embedFunctions.js' which contains the logic to generate embeddings
+const { InitiateEmbeddings } = require('./embedFunctions');
+require('dotenv').config();
+
+const appInsights = require('applicationinsights');
+
+// Configure Application Insights with your instrumentation key or connection string
+const instrumentationKey = process.env.APPINSIGHTS_INSTRUMENTATIONKEY;
+
+// Or use connection string
+const connectionString = process.env.APPINSIGHTS_CONNECTIONSTRING;
+
+appInsights.setup(connectionString || instrumentationKey)
+    .setAutoCollectRequests(true)
+    .setAutoCollectPerformance(true, true)
+    .setAutoCollectExceptions(true)
+    .setAutoCollectDependencies(true)
+    .setAutoCollectConsole(true, true)
+    .setUseDiskRetryCaching(true)
+    .start();
+
+const client = appInsights.defaultClient;
+
+// Define an Azure Function triggered by Blob storage events using Event Grid
+app.storageBlob('BlobTriggerEventGrid', {
+    path: 'samples-workitems/{name}', // The path to the Blob storage container, with a placeholder for the blob name
+    source: 'EventGrid', // The source of the events is Event Grid
+    connection: 'c0008c_STORAGE', // The name of the connection string to the Azure Storage account
```

Comment:
Storage name should be from env file.

Reply:
We are reading the configuration from .env file only.
![image](https://github.com/user-attachments/assets/98440454-faa9-4ad6-a351-abeac8072c55)

This storage name is for testing the Azure function locally.

Reply:
Also, update the Readme to replace these values manually.